be/src/exec/tablet_sink.cpp

```diff
@@ -612,8 +616,17 @@ Status OlapTableSink::close(RuntimeState* state, Status close_status) {
         COUNTER_SET(_validate_data_timer, _validate_data_ns);
         COUNTER_SET(_wait_in_flight_packet_timer, _wait_in_flight_packet_ns);
         COUNTER_SET(_serialize_batch_timer, _serialize_batch_ns);
-
         state->update_num_rows_load_filtered(_number_filtered_rows);
+
+        // print log of add batch time of all node, for tracing load performance easily
+        std::stringstream ss;
+        ss << "finished to close olap table sink. load_id=" << print_id(_load_id)
+                << ", txn_id=" << _txn_id << ", node add batch time(ns): ";
+        for (auto const& pair: _node_add_batch_time_map) {
+            ss << "{" << pair.first << "=" << pair.second << "}";
+        }
+        LOG(INFO) << ss.str();
```

Comment:
I think you should VLOG(), and use be.conf to switch it to open.

and you should use if (LOG.level) to enclose string constructor.

Reply:
VLOG can not be used online..
I think add one log for one load job is OK.

Reply:
You can use pattern to make only this module

like http://rpg.ifi.uzh.ch/docs/glog.html#verbose